src/modules/settlement/breezekiln/breezekiln.tscn

```diff
@@ -21,12 +24,6 @@ label = "Visit the Wind Altar"
 scene_path = "res://modules/settlement/breezekiln/wind_altar/wind_altar.tscn"
 metadata/_custom_type_script = "uid://dx8li0wjwcoe3"
 
-[sub_resource type="Resource" id="Resource_572gp"]
-script = ExtResource("2_ydiv2")
-label = "Play Fire Fighters Minigame"
-scene_path = "res://modules/minigame/fire_fighters/fire_fighters.tscn"
-metadata/_custom_type_script = "uid://dx8li0wjwcoe3"
-
```

Comment:
I removed the placeholder button for "Play Fire Fighters Minigame" because it doesn't set up the settlement context. So now, there is a debug UI button that does the same thing (or press "f" to launch).